Registration Campaign GA Docs

mjsantani opened the pull request.

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md:
```diff
@@ -0,0 +1,52 @@
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
```
GLOBAL: (Please apply this change to all resource topics) 

- Please, remove the beta disclaimer as this is a v1.0 article.

JarbasHorst commented on changelog/Microsoft.DirectoryServices.json:
```diff
@@ -9235,6 +9235,64 @@
+          "ApiChange": "Property",
+          "ChangedApiName": "registrationEnforcement",
+          "ChangeType": "Addition",
+          "Description": "Added the **registrationEnforcement** property to [authenticationMethodsPolicy](https://docs.microsoft.com/en-us/graph/api/resources/authenticationMethodsPolicy?view=graph-rest-1.0) resource.",
```
Please, add "the" before "authenticationMethodsPolicy". Suggested revision:

- Added the \*\*registrationEnforcement\*\* property to the \[authenticationMethodsPolicy\]...

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md:
```diff
@@ -0,0 +1,52 @@
+      "@odata.type": "microsoft.graph.authenticationMethodsRegistrationCampaignIncludeTarget"
+    }
+  ],
+  "snoozeDurationInDays": "Integer",
```
For consistency, please replace "Integer" with "Int32".

JarbasHorst commented on api-reference/v1.0/resources/registrationenforcement.md:
```diff
@@ -0,0 +1,40 @@
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Enforce registration at sign-in time. This can currently only be used to remind users to set up targeted authentication methods (Microsoft Authenticator) using the 'authenticationMethodsRegistrationCampaign`.
```
Please, fix the backtick used to highlight "authenticationMethodsRegistrationCampaign". Suggested revision:

- using the \`authenticationMethodsRegistrationCampaign\`.

JarbasHorst replied:
For clarity, please add "for example," before Microsoft Authenticator. Suggested revision:

- ...(for example, Microsoft Authenticator)...

JarbasHorst commented on api-reference/v1.0/resources/registrationenforcement.md:
```diff
@@ -0,0 +1,40 @@
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|authenticationMethodsRegistrationCampaign|[authenticationMethodsRegistrationCampaign](../resources/authenticationmethodsregistrationcampaign.md)|Run campaigns to remind users to setup targeted authentication methods.|
```
Please, replace "setup" (noun) with "set up" (verb).

JarbasHorst commented on api-reference/v1.0/resources/excludetarget.md:
```diff
@@ -0,0 +1,40 @@
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|The object identifier of an Azure AD user or group.|
```
Before using acronyms, I recommend you to spell out the words first followed by the acronym in parentheses. However, in this case, I recommend you to just spell out Azure AD. Suggested revision:

- The object identifier of an Azure Active Directory user or group.

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md:
```diff
@@ -0,0 +1,42 @@
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|id|String|The object identifier of an Azure AD user or group.|
```
Before using acronyms, I recommend you to spell out the words first followed by the acronym in parentheses. However, in this case, I recommend you to just spell out Azure AD. Suggested revision:

- The object identifier of an Azure Active Directory user or group.

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md:
```diff
@@ -0,0 +1,52 @@
+|:---|:---|:---|
+|excludeTargets|[excludeTarget](../resources/excludetarget.md) collection|Users and groups of users that are excluded from being prompted to set up the authentication method.|
+|includeTargets|[authenticationMethodsRegistrationCampaignIncludeTarget](../resources/authenticationmethodsregistrationcampaignincludetarget.md) collection|Users and groups of users that are prompted to set up the authentication method.|
+|snoozeDurationInDays|Int32|Specifies the number of days that the user sees a prompt again if they select "Not now" and snoozes the prompt. Minimum 0 days. Maximum: 14 days. If the value is “0” – The user is prompted during every MFA attempt.|
```
For consistency, please add colon after Minimum. Suggested revision:

- Minimum: 0 days...

JarbasHorst replied:
For consistency, please replace these quotation marks “” with "". Also, please remove the dash and combine the sentences with a comma. Suggested revision:

- If the value is "0", the user is prompted during every MFA attempt.

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md:
```diff
@@ -0,0 +1,52 @@
+|excludeTargets|[excludeTarget](../resources/excludetarget.md) collection|Users and groups of users that are excluded from being prompted to set up the authentication method.|
+|includeTargets|[authenticationMethodsRegistrationCampaignIncludeTarget](../resources/authenticationmethodsregistrationcampaignincludetarget.md) collection|Users and groups of users that are prompted to set up the authentication method.|
+|snoozeDurationInDays|Int32|Specifies the number of days that the user sees a prompt again if they select "Not now" and snoozes the prompt. Minimum 0 days. Maximum: 14 days. If the value is “0” – The user is prompted during every MFA attempt.|
+|state|advancedConfigState|Enable or disable the feature. Possible values are: `default`, `enabled`, `disabled`, `unknownFutureValue`. The `default` value is used when the configuration hasn't been explicitly set and uses the default behavior of Azure AD for the setting. The default value is `disabled`.|
```
For clarity, I recommend you to spell out Azure AD. Suggested revision:

- ...the default behavior of Azure Active Directory for the setting...

opbld34 commented:
Docs Build status updates of commit _[de7bdb8](https://github.com/microsoftgraph/microsoft-graph-docs/commits/de7bdb8f18b85bdebd740501785ccf9e174401d8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111041803418138-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-05T18%3A05%3A29Z&sr=b&sp=r&sig=Xj3LJS9gWAlCSV9WT%2BcCXlYumUFVi2AwT8RVv%2FQGr9s%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mjsantani commented:
@JarbasHorst thank you for the incredibly detailed review. I've addressed all comments - please do take a look.

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodspolicies-overview.md:
```diff
@@ -26,6 +27,11 @@ The authentication method policies APIs are used to manage policy settings. For
 |[microsoftauthenticatorauthenticationmethodconfiguration](microsoftauthenticatorauthenticationmethodconfiguration.md)|Define users who can use Microsoft Authenticator on the Azure AD tenant.|
 |[emailauthenticationmethodconfiguration](emailauthenticationmethodconfiguration.md)|Define users who can use email OTP on the Azure AD tenant.|
 
+## Policies available for authentication methods Registration Campaign:
```
"Registration Campaign" doesn't need to be capitalized. Please replace "Registration Campaign" with "registration campaign".

JarbasHorst commented on api-reference/v1.0/resources/authenticationmethodspolicies-overview.md:
```diff
@@ -26,6 +27,11 @@ The authentication method policies APIs are used to manage policy settings. For
+## Policies available for authentication methods Registration Campaign:
+|Policy       | Description |
+|:---------------------------|:------------|
+|[authenticationMethodsRegistrationCampaign](authenticationmethodsregistrationcampaign.md)| Define users who should be reminded to set up an authentication method (only supported for the Microsoft Authenticator).|
```
If the information in the parentheses applies to the authentication method, no changes are required. Now, if "only supported for the Microsoft Authenticator" applies to the policy "authenticationMethodsRegistrationCampaign", I recommend you to remove this information from the parentheses and create a new sentence. Suggested revision:

- ...to set up an authentication method. Only supported for the Microsoft Authenticator.

mjsantani replied:
Made an update and added "Currently".

JarbasHorst replied:
Hi @mjsantani, I don't see this change. Can you please check if it was committed correctly?

mjsantani commented:
@JarbasHorst - Addressed feedback. Please take a look :)

opbld33 commented:
Docs Build status updates of commit _[15e7327](https://github.com/microsoftgraph/microsoft-graph-docs/commits/15e73277ff609eb37ef30d7bcce619f828e7f6bc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C5%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111051925240376-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-06T19%3A26%3A54Z&sr=b&sp=r&sig=bS7yyavSpeo2RFMukkMpO7acjU6G0Hynh%2BohwAqTG3Y%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld34 commented:
Docs Build status updates of commit _[fc09fae](https://github.com/microsoftgraph/microsoft-graph-docs/commits/fc09fae971a265e0130642fec45baa6c7fedc955)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111130022228230-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-14T00%3A24%3A22Z&sr=b&sp=r&sig=Bla5MXH98ev03LmkwwB1NuXaV0%2BosjTeGrhoUc339do%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld34 commented:
Docs Build status updates of commit _[bb4fedd](https://github.com/microsoftgraph/microsoft-graph-docs/commits/bb4feddd61fb51517d881cce6e01c21c8d2a330a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111130027581000-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-14T00%3A30%3A09Z&sr=b&sp=r&sig=oCasFWDsF%2BNuQ49hihcLDa4EyuqCt%2FGqtOFl%2BLAKTuQ%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mjsantani commented:
@JarbasHorst - Can we get this approved since all changes have been addressed? I need these to be merged ASAP since the feature is going to be GAing early next week.

Lauragra commented:
> @JarbasHorst - Can we get this approved since all changes have been addressed? I need these to be merged ASAP since the feature is going to be GAing early next week.

@mjsantani , I see one remaining comment from Jarbas that might not be resolved - please double check? When your feature is GA, please set the ready to merge label on this PR and we'll merge and publish. We don't publish the docs before the feature is available.

opbld32 commented:
Docs Build status updates of commit _[783d2c2](https://github.com/microsoftgraph/microsoft-graph-docs/commits/783d2c272e6cc88cb249fe577ad6fe78cbecba02)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C13%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111130141391608-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-14T01%3A43%3A15Z&sr=b&sp=r&sig=kvK0cMKn%2BWL2o0ZrRWxjVA3BumjZhRpeJz3I58CapNw%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

mjsantani commented:
@Lauragra thanks for catching that! I've since fixed the PR and will set the right labels. To confirm, we don't need the second approval before merging?

mjsantani commented:
@Lauragra @JarbasHorst the schema is live, I've addressed the comments, and changed the labels to "schema live" and "ready to merge". 

Let me know if you need anything else from me to wrap this one up. Thanks!

opbld30 commented:
Docs Build status updates of commit _[16a3245](https://github.com/microsoftgraph/microsoft-graph-docs/commits/16a324564711fb54729883f98ad83a6248284629)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C16%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111160344360932-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-17T03%3A46%3A09Z&sr=b&sp=r&sig=5bj7QKGzW3kIELzrO7Cag5cQbVEWnHXI0zd3fatnRRY%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[ab6445a](https://github.com/microsoftgraph/microsoft-graph-docs/commits/ab6445aa46e351264ff85003a744ee06a1567add)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/authenticationmethodspolicy-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-get.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-get?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/api/authenticationmethodspolicy-update.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/api/authenticationmethodspolicy-update.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/authenticationmethodspolicy-update?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicies-overview.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicies-overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicies-overview?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodspolicy.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodspolicy.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodspolicy?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaign.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaign?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/authenticationmethodsregistrationcampaignincludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/authenticationmethodsregistrationcampaignincludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/excludetarget.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/excludetarget.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/excludetarget?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[api-reference/v1.0/resources/registrationenforcement.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/api-reference/v1.0/resources/registrationenforcement.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/registrationenforcement?view=graph-rest-1.0&branch=pr-en-us-14765) (>= graph-rest-1.0) |
[changelog/Microsoft.DirectoryServices.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/2021-10-27-RegistrationCampaign/changelog/Microsoft.DirectoryServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C16%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202111160345575185-14765%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-17T03%3A48%3A28Z&sr=b&sp=r&sig=Wr%2FYfHLFqPPfG%2FNkbkDtqgD8ScfcFK%2BRyUX6gd9ZV5s%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
